rpm/0052-sailfishos-gecko-Make-TextureImageEGL-hold-a-referen.patch

```diff
@@ -1,4 +1,4 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From a642f9e2ebd9a862de0cee7e9e77d33ab9547ef7 Mon Sep 17 00:00:00 2001
```

Comment:
Should be formatted like...

> git format-patch --zero-commit  -N sha1

That said just zeroing this would be enough.

Reply:
Thanks; I've zeroed it out.